### PR TITLE
Updates for 3.7

### PR DESCRIPTION
**Pull request by sfshaza2**
Primarily video and what's new pages.

### CODE REVIEW

File: src/whats-new.md [hunk 1]
```diff
@@ -178,6 +176,10 @@ to reflect the latest available codelabs.
 We've published the following articles on the [Flutter Medium][]
 publication since the last stable release:
 
+* [What's next for Flutter][]
```
Comment (domesticmouse) [inline src/whats-new.md:179]: I don't see the URL for this below. Was it already in the content?

**Review by domesticmouse — APPROVED** [review]

File: src/whats-new.md (hunk 1 above)
Comment (domesticmouse) [reply src/whats-new.md:179]:
```console
Checking for valid link references...
check_link_references: Invalid link references found!

whats-new/index.html:
[What’s new in Flutter 3.7][]
make[1]: *** [Makefile:120: build-image-and-check-links] Error 1
make[1]: Leaving directory '/home/runner/work/website/website'
make: *** [Makefile:136: build] Error 2
```

Comment (sfshaza2) [reply src/whats-new.md:179]: yup!

**Review by parlough — APPROVED** [review]
I updated the submodules referenced in the branch so they aren't downgraded on merge. Looks good to me :)

**Pull request merged**